website/docs/patterns/configuring_plugins.md

```diff
@@ -0,0 +1,186 @@
+---
+id: configuring_plugins
+title: Configuring Plugins
+---
+
+import {ExampleGithubLink} from "@site/src/components/GithubLink"
+
+Hydra plugins usually comes with sensible defaults which works with minimal configuration.
+There are two primary ways to customize the configuration of a plugin:
+- Overriding it directly in your primary config
+- Extending the config and using it from your primary config.
+	
+The first method is the simplest, but it makes it harder to switch to a different plugin configuration.
+The second method is a bit more complicated, but makes it easy to switch between different plugin configurations.
+
+
+The following methods apply to all Hydra plugins. In the following examples, we will configure a imaginary Launcher plugin
+`MoonLauncher`. The Launcher has two modes: `falcon9`, which actually launches the application to the Moon and 
+`simulation` which simulates a launch.
+
+The config schema for MoonLauncher looks like:
+
+
+
+<div className="row">
+<div className="col col--6">
+
+```python
+@dataclass
+class Falcon9Conf:
+  ton_fuel:  int = 10
+
+
+
+
+```
+</div>
+<div className="col  col--6">
+
+```python
+@dataclass
+class Simulation:
+  ton_fuel:  int = 10
+  window_size:
+    width: 1024
+    height: 768
+
+```
+</div>
+</div>
+
+
+
+### Overriding in primary config
+We can directly override Launcher config in primary config.
+
+<div className="row">
+<div className="col col--4">
+
+```yaml title="config.yaml" 
+a: 1
+
+hydra:
+  launcher:
+    ton_fuel: 2
+
+
+
+
+
+
+
+
+
+```
+</div>
+<div className="col col--4">
+
+```commandline title="command-line override" 
+hydra/launcher=falcon9
+
+
+```
+```yaml title="resulting launcher config"  {3}
+hydra:
+  launcher:
+    ton_fuel: 2
+
+
+
+```
+</div>
+<div className="col col--4">
+
+```commandline title="command-line override" 
+hydra/launcher=simulation
```

Comment:
This has a horizontal scrollbar.
Try to rename the config name for simluation to something shorter (sim?) or to reorgranize the page a bit to give this column more space.